static/app/views/issueList/index.tsx

```diff
@@ -103,19 +104,23 @@ function IssueViewWrapper({children}: Props) {
     return <NotFound />;
   }
 
-  return (
-    <SentryDocumentTitle title={groupSearchView?.name} orgSlug={organization.slug}>
```

Comment:
The main problem was here - we shouldn't render this component at all if there is no view loaded.